Turkey/Program.cs

```diff
@@ -37,12 +37,20 @@ public class Program
             Arity = ArgumentArity.ZeroOrMore
         };
 
+        public static readonly Option<IEnumerable<string>> testsOption = new Option<IEnumerable<string>>(
+            new string[] { "--tests" },
```

Comment:
suggestion: Could we rename this to `test-name`? This way we have the same argument name as [autopkgtest(1)](https://manpages.ubuntu.com/manpages/noble/en/man1/autopkgtest.1.html). Additionally the plural is counter intuitive if you have to do `--tests name1 --tests name2`. 
```suggestion
            new string[] { "--test-name" },
```

Reply:
Good point. Changing that.

Reply:
Setting this to resolved with e3f9d37.